podman-compose.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Podman Compose
+sidebar: home_sidebar
+permalink: /podman-compose/
+toc: false
+---
+
+Many members of the community have requested a production-ready container to deploy Pulp.
+In an effort to address this feedback, we have re-used images from [Pulp Operator](/pulp-operator/) to provide a podman compose.
+Because this podman compose is new, so far it has received limited feedback and testing.
+We want to hear from you. Let us know what you think on our [community discourse](https://discourse.pulpproject.org/).
+
+To use Pulp's podman compose, you must have an understanding of podman compose and also how to configure Pulp's podman compose to best suit your deployment requirements.
+At the moment, there is minimal documentation for Pulp's podman compose.
+If you'd like to add some tips, tricks, or tutorials, feel free to raise a PR against [this page](https://github.com/pulp/pulpproject.org/).
+
+Pulp's podman compose requires requires podman 3 or higher, as well as podman-compose 0.1 or higher.
```

Comment:
Actually, it should be 3.x and 0.1.z (higher versions break it)
https://github.com/pulp/pulp-operator/tree/main/containers#compose
and you can point users to this link ^

Reply:
Interesting. I completely misread your statement! Thanks.

Reply:
```suggestion
Pulp's podman compose requires podman 3 or higher, as well as podman-compose 0.1 or higher.
```

Reply:
Fixed x 2